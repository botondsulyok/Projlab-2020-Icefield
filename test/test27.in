field spawn 0
entity master1 Eskimo spawn
entity master2 Eskimo spawn
entity master3 Eskimo spawn
entity master4 Eskimo spawn
item pist1 Pistol master1
item pist2 Pistol master2
item pist3 Pistol master3
item pist4 Pistol master4
item cart1 Cartridge master1
item cart2 Cartridge master2
item cart3 Cartridge master3
item cart4 Cartridge master4
item light1 Light master1
item light2 Light master2
item light3 Light master3
item light4 Light master4
stats master1
stats master2
stats master3
stats master4
assemble master1
stats spawn
stats master1
stats master2
stats master3
stats master4