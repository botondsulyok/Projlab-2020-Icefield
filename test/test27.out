id: master1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1, cart1, light1

id: master2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: pist2, cart2, light2

id: master3
type: eskimo
iceUnit: spawn
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: pist3, cart3, light3

id: master4
type: eskimo
iceUnit: spawn
movebehaviour: walk3
bodyheat: 5
stamina: 4
digger: barehand3
waterProtecter: lung3
pullerDevice: basicpull3
tent: null
components: pist4, cart4, light4

Game Won!
id: spawn
type: iceunit
building: plains0
snow: 0
pickable: null
standers: master1, master2, master3, master4
pullers: master1, master2, master3, master4
masters: master1, master2, master3, master4
neighbours: null

id: master1
type: eskimo
iceUnit: spawn
movebehaviour: walk0
bodyheat: 5
stamina: 4
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: pist1, cart1, light1

id: master2
type: eskimo
iceUnit: spawn
movebehaviour: walk1
bodyheat: 5
stamina: 4
digger: barehand1
waterProtecter: lung1
pullerDevice: basicpull1
tent: null
components: pist2, cart2, light2

id: master3
type: eskimo
iceUnit: spawn
movebehaviour: walk2
bodyheat: 5
stamina: 4
digger: barehand2
waterProtecter: lung2
pullerDevice: basicpull2
tent: null
components: pist3, cart3, light3

id: master4
type: eskimo
iceUnit: spawn
movebehaviour: walk3
bodyheat: 5
stamina: 4
digger: barehand3
waterProtecter: lung3
pullerDevice: basicpull3
tent: null
components: pist4, cart4, light4

